src/ducks/components/App.jsx

```diff
@@ -16,13 +16,22 @@ import { Layout, Main } from 'cozy-ui/react/Layout'
 import Alerter from 'cozy-ui/react/Alerter'
 import { IconSprite } from 'cozy-ui/react'
 
+import { enabledStoreParts } from 'config'
+
+const componentsMap = {
+  discover: Discover,
+  myapps: MyApplications
+}
+
 export class App extends Component {
   constructor(props) {
     super(props)
     props.initApp() // fetch apps without icons
   }
 
   render() {
+    const parts = enabledStoreParts || ['discover', 'myapps']
```

Comment:
This information could be passed as props IMHO.

How I see this:
* The `<Root />` component have a dependency to the configuration and inject it into the `<App />` component (it seems logical that an app should have a config property).
* The `<App />` then use `config` to inject props into its children components

Example:
**Root.jsx**
```jsx
<App config={config} />
```

**App.jsx**
```jsx
  render(){
    const { config } = this.props
    const sections = config.enabledSections || ['discover', 'myapps' ]
   // ...
   <Sidebar sections={sections} />
   // ...
   { sections.map(name =>
   // ...
  }
```

Reply:
Using props means that we will have to pass this config props to each children, I am not sure it will be better to undertand :/

Reply:
HOC seems a good idea 👍